layers/+source-control/version-control/funcs.el

```diff
@@ -59,7 +59,7 @@
   (let ((current-prefix-arg t))
     (call-interactively
      (cl-case version-control-diff-tool
-       (diff-hl     'diff-hl-margin-minor-mode)
+       (diff-hl     'diff-hl-margin-mode)
```

Comment:
@StreakyCobra This is the docstring for `diff-hl-margin-minor-mode`

  "Toggle displaying `diff-hl-mode' highlights on the margin locally.
You probably shouldn't use this function directly."

It's not autoloaded, which seems to be on purpose. Any idea why this function is used and not the plain mode?